src/bin/bat/config.rs

```diff
@@ -15,6 +16,46 @@ pub fn config_file() -> PathBuf {
         .unwrap_or_else(|| PROJECT_DIRS.config_dir().join("config"))
 }
 
+pub fn generate_config_file() {
+    let config_file = config_file();
+    if config_file.exists() {
+        println!("A config file already exists at: {}", config_file.to_string_lossy());
+
+        print!("Overwrite? (y/n): ");
+        let _ = io::stdout().flush();
```

Comment:
If you change this function to return `Result<()>`, you can use
```rust
io::stdout().flush()?;
```
here. Another alternative would be `io::stdout().flush().ok()`.